graphx/src/main/scala/org/apache/spark/graphx/lib/ConnectedComponents.scala

```diff
@@ -37,7 +37,7 @@ object ConnectedComponents {
    */
   def run[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED]): Graph[VertexId, ED] = {
     val ccGraph = graph.mapVertices { case (vid, _) => vid }
-    def sendMessage(edge: EdgeTriplet[VertexId, ED]) = {
+    def sendMessage(edge: EdgeTriplet[VertexId, ED]): Iterator[(VertexId, VertexId)] = {
```

Comment:
This change is fine, but hopefully the rule doesn't require it -- `sendMessage` isn't a public method here.